frontend/src/store/store.ts

```diff
@@ -1456,6 +1456,10 @@ export default new Vuex.Store<IState>({
           { feeMultiplier: num * 4 * chosenElementFee * slippageMultiplier, allowInGameOnlyFunds: true }
         );
 
```

Comment:
This will need updating to handle the pay first, pop second nature of the flow. Would like to note we will need some visual feedback for the player to know they have a weapon waiting if for whatever reason the payment succeeds but they cancel/fail the claim. This is made more complicated by the fact that the seeds are separated out by amounts. Not sure what's the best approach there. Calling hasSeed 10 times may be a little slow for the RPC to respond, but including that behavior in the solidity function will add to size. I don't have a preference.

Worth mentioning this behavior will have to be updated in the bazaar repo too. (but at least there it's 1 shield at a time)